_docs/connect-a-data-source/plugins/110-s3-storage-plugin.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "S3 Storage Plugin"
+parent: "Connect a Data Source"
+---
+Drill works with data stored in the cloud. With a few simple steps, you can configure the S3 storage plugin for Drill and be off to the races running queries.
+
+## Connecting Drill to S3
+
+Starting with version 1.3.0, Drill has the ability to query files stored on Amazon's S3 cloud storage using the S3a library. This is important, because S3a adds support for files bigger than 5 gigabytes (these were unsupported using Drill's previous S3n interface).
+
+There are two simple steps to follow: (1) provide your AWS credentials (2) configure S3 storage plugin with S3 bucket
+
+#### (1) AWS credentials
+
+To enable Drill's S3a support, edit the file conf/core-site.xml in your Drill install directory, replacing the text ENTER_YOUR_ACESSKEY and ENTER_YOUR_SECRETKEY with your AWS credentials.
+
+```
+<configuration>
+
+  <property>
+    <name>fs.s3a.access.key</name>
+    <value>ENTER_YOUR_ACCESSKEY</value>
+  </property>
+
+  <property>
+    <name>fs.s3a.secret.key</name>
+    <value>ENTER_YOUR_SECRETKEY</value>
+  </property>
```

Comment:
Do we need to add a note about the threading property that you have played with changing from 4 to 100?